framer/Utils.coffee

```diff
@@ -434,6 +434,14 @@ Utils.deviceFont = (os) ->
 	return "Segoe UI" if os is "windows"
 	return "Helvetica"
 
+Utils.webFont = (font) ->
+	link = document.createElement("link")
+	link.href = "https://fonts.googleapis.com/css?family=#{font}"
+	link.rel = "stylesheet"
+	document.getElementsByTagName("head")[0].appendChild(link)
+
+	return font
```

Comment:
Hmm, googleFont sounds ok to me, although I don't like that google is sometimes also used as a verb (are we googling for a font in this method?). More important than the naming is my last remark though: we add something to the document every time this function runs, which probably is on every reload. This is not ok, so we should keep a cache of which fonts are already loaded and which aren't.

What I meant by returning is: What are you using the return value of webFont for? Why not use the variable that you pass into webfoot for that?